schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Thunderbird In-App Notifications",
+  "type": "array",
+  "items": [
+    {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Unique ID set by the server"
+        },
+        "start_at": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp after which Thunderbird will show the event after startup."
+        },
+        "end_at": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp after which Thunderbird will never show the event, even if it has never been shown to the user."
+        },
```

Comment:
I assume these will be `...T...Z` timestamps? From the json schema docs I could find they didn't specify that it should include a timezone.

Reply:
I think that including a timezone should be fine?
I couldn't find a validator that actually checked if a `date-time` string field was formatted correctly.

(The validators said everything was :+1: whether I put in a timestamp with/without a timezone...and even the string `"this is not a timestamp"`)

Reply:
This should always be UTC for consistency when we are publishing events. It's fine if the client is able to handle other time zones, but not required.